src/features/discography/components/discography.module.scss

```
@@ -0,0 +1,55 @@
+@import "../../../styles/_variables";
```

Stylelint (17.10.0)

[error] Unexpected leading underscore in imported partial name (scss/load-no-partial-leading-underscore)

(scss/load-no-partial-leading-underscore)